dcim_devices.go

```diff
@@ -15,6 +15,7 @@
 package netbox
 
 // Device is a network device.
+// At this time, the Device type is not implemented.
```

Comment:
What's the rationale behind adding this comment?  The `Device` type is indeed implemented.